static/css/global.css

```
@@ -185,6 +185,26 @@ footer {
   opacity: 0.5;
 }
 
+.pagination-number-button {
+  background-color: #236BE4;
+  border-color: #236BE4;
```

Solución:
```suggestion
  border: 1px solid var(--blue);
```
...borrando la línea 199 o reemplazándolo ahí